configuration/pih/liquibase/liquibase.xml

```diff
@@ -643,5 +643,19 @@
         <comment>Remove old authentication events following bug fixes</comment>
         <delete tableName="authentication_event_log"/>
     </changeSet>
-
+        
+    <changeSet id="20221101-initial-create-of-petl-load-times-table" author="ddesimone">
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="0">
+                select table_exists("petl_load_times");
```

Comment:
Better to follow existing standard and conventions here.  See where I add the authentication_event_log table.  Something like:

```<not><tableExists tableName="petl_load_times"/></not>```